src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2935,7 +2935,7 @@ void processContextElement
   std::map<std::string, std::string>&  uriParams,   // FIXME P7: we need this to implement "restriction-based" filters
   const std::string&                   xauthToken,
   const std::string&                   apiVersion,
-  bool                                 checkEntityExistance
+  const std::string&                   ngsiv2Flavour
```

Comment:
At the end, checkEntityExistance was a bit limitted to achieve specific functionality at mongoBackend update for some update cases, so I opted for thiw "flavour" thing.

NTC